driver_test.clj

```diff
@@ -0,0 +1,116 @@
+(ns metabase.driver-test
```

Comment:
Can't you just replace the failing tests, or do you have to copy the entire file?

Reply:
I've tried to override the tests to no avail